packages/ui/src/theme/css/component/inAppMessaging/backdrop.scss

```diff
@@ -0,0 +1,11 @@
+.amplify-inappmessaging-backdrop {
+  align-items: center;
+  background-color: var(--amplify-colors-overlay-50);
+  bottom: 0;
+  justify-content: center;
+  left: 0;
+  position: fixed;
+  right: 0;
+  top: 0;
+  z-index: 1000;
```

Comment:
I think I'd like to see z-index as token as well. Just thinking that 1000 is a common number people use to "win" the z-index wars, which means it could potentially conflict more often.